### PR TITLE
Diagram the Location of a Command

### PR DESCRIPTION
**Pull request by SvetlanaMikheeva**

### CODE REVIEW

File: api-reference/50 Common/Object Structures/dxDiagramCustomCommand/location.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ default: 'before'
...
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a location for the command or separator on the main toolbar.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+[note]
+This property does not allow you to set the command's or separator's location in the context menu, history and view toolbars.
```
Comment (albertov05) [inline api-reference/50 Common/Object Structures/dxDiagramCustomCommand/location.md]: Is "history" also a toolbar? If so, then "history and view toolbars" should be "and history and view toolbars".
Also, I'd like to change "command's or separator's location" to "location of the command or separator".

```suggestion
This property does not allow you to set the location of the command or separator in the context menu, and history and view toolbars.
```

File: concepts/05 UI Components/Diagram/50 How To/20 Customize the Diagram Component/15 Specify a Command's Location on the Main Toolbar.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+- **center**  
+ Places the command in the center of the main toolbar.
+
+- **before**  
+ Places the command before the central element(s).
+
+- **after**  
+ Places the command after the central element(s).
+
+The Diagram UI component displays commands with identical location in the order in which you listed them. When the main toolbar cannot accommodate all commands, the component places excess commands in the overflow menu.
```
Comment (albertov05) [inline concepts/05 UI Components/Diagram/50 How To/20 Customize the Diagram Component/15 Specify a Command's Location on the Main Toolbar.md]: What do you mean by "commands with identical location"?
Is it possible for two different commands to have the same location?

Comment (SvetlanaMikheeva) [reply concepts/05 UI Components/Diagram/50 How To/20 Customize the Diagram Component/15 Specify a Command's Location on the Main Toolbar.md]: "commands with identical location" = "commands with the same **location** property value". For instance:

`commands: [ { name: "copy", location: "center" },  { name: "paste", location: "center" }, ]`

Comment (SvetlanaMikheeva) [reply concepts/05 UI Components/Diagram/50 How To/20 Customize the Diagram Component/15 Specify a Command's Location on the Main Toolbar.md]: Can we say the following?

```suggestion
The Diagram UI component displays commands with the same value of the **location** property in the order in which you listed them. When the main toolbar cannot accommodate all commands, the component places excess commands in the overflow menu.
```

Comment (albertov05) [reply concepts/05 UI Components/Diagram/50 How To/20 Customize the Diagram Component/15 Specify a Command's Location on the Main Toolbar.md]: Thanks for the clarification. How about the following?

```suggestion
The Diagram UI component displays commands with the same **location** property value in the order in which you listed them. When the main toolbar cannot accommodate all commands, the component places excess commands in the overflow menu.
```

**Review by AnnaKondratova — APPROVED** [review]

**Pull request merged**